src/inspector_agent.cc

```diff
@@ -53,7 +53,51 @@ void PrintDebuggerReadyMessage(int port) {
 }
 
 bool AcceptsConnection(inspector_socket_t* socket, const std::string& path) {
-  return 0 == path.compare(0, sizeof(DEVTOOLS_PATH) - 1, DEVTOOLS_PATH);
+  return StringEqualNoCaseN(path.c_str(), DEVTOOLS_PATH,
+                            sizeof(DEVTOOLS_PATH) - 1);
+}
+
+#ifdef _MSC_VER
+std::string AbsolutePath(const char* path) {
+  bool is_relative;
+  if (path && path[0]) {
+    char *file_name;
+    char buf[MAX_PATH * 2];
+    size_t len = GetFullPathName(path, sizeof(buf), buf, &file_name);
+
+    if (len && len < sizeof(buf))
+      return std::string(buf, len);
+  }
+  return std::string();
+}
+#else  // Non-windows
+std::string AbsolutePath(const char* path) {
+  char PATH_SEPARATOR = '/';
+  bool is_relative;
+  if (path == nullptr || path[0] == '\0') {
+    return std::string();
+  } else {
+    is_relative = path[0] != PATH_SEPARATOR;
+  }
+  if (is_relative) {
+    char buf[PATH_MAX];
+    size_t size = sizeof(buf);
+    if (uv_cwd(buf, &size) != 0) {
+      return std::string(path);
+    }
+    return std::string(buf, size) + PATH_SEPARATOR + path;
+  } else {
+    return std::string(path);
+  }
+}
+#endif  // defined(_MSC_VER)
```

Comment:
I might be mistaken but it looks like this function could be implemented in terms of `uv_fs_realpath()`?